test/integration/cx_limit_overload_integration_test.cc

```diff
@@ -0,0 +1,185 @@
+#include "envoy/config/bootstrap/v3/bootstrap.pb.h"
+#include "envoy/config/overload/v3/overload.pb.h"
+
+#include "test/integration/integration.h"
+#include "test/test_common/test_runtime.h"
+
+namespace Envoy {
+
+namespace {
+
+envoy::config::overload::v3::OverloadManager overloadManagerProtoConfig(uint32_t max_cx) {
```

Comment:
consider calling this `generateMaxConnectionsOverloadConfig` or similar.

I think proto is already implied given the return type